privilege-management.md

```diff
@@ -181,34 +184,62 @@ SHOW GRANTS FOR `rw_user`@`192.168.%`;
 +------------------------------------------------------------------+
 ```
 
+### Dynamic privileges
+
+<!-- include a "since" for the version number after TiDB 5.0 --> TiDB features support dynamic privileges, a feature borrowed from MySQL 8.0. Dynamic privileges are intended to replace the `SUPER` privilege by implementing more fine-grained access to certain operations. For example, with dynamic privileges it is possible to grant access to perform `BACKUP` and `RESTORE` operations but not `SHOW PROCESSLIST`.
```

Comment:
+ Can we say "Since v5.1" in the upcoming release-5.1 branch but remove it in master (because master is the latest by default)?
+ Could you specify "WHO" can grant access to perform `BACKUP` and `RESTORE`? With this information, the translated Chinese text will look more natural.
```suggestion
TiDB supports dynamic privileges, a feature borrowed from MySQL 8.0. Dynamic privileges are intended to replace the `SUPER` privilege by implementing more fine-grained access to certain operations. For example, with dynamic privileges enabled, it is possible to access to perform `BACKUP` and `RESTORE` operations but not `SHOW PROCESSLIST`.
```

Reply:
We can say since 5.1 - I was waiting for the versoin to be named.  I'll change it.

It should read:
"For example, with dynamic privileges ~enabled~, it is possible to grant.. "

Dynamic privileges are always enabled. The difference is if you chose to allocate a fine-grained privileges or not. Both are possible.

> Could you specify "WHO" can grant access to perform BACKUP and RESTORE? With this information, the translated Chinese text will look more natural.

It is possible for a system administrator to create a limited privilege account for use only in backup and restore operations. The backup and restore user might be part of a scripted process  - by limiting its access you help reduce the impact of privilege escalation scenarios (it is true for both scripted processes and users - you should always limit access to principal of least privilege.)